### DISCUSSION TITLE
CronJob with timezone

### DISCUSSION THREAD

**Discussion by fiddep** [quirrel-dev/quirrel#829] [Q&A] [answered]
Hi!

Tried out running cron jobs with specified time zone now that I saw it was supported.
I have a problem that when running a job `["0 6 * * 1-5", "Europe/Stockholm"]` the time zone does not seem to be honored and it's executed UTC time instead, so 08:00 Swedish time instead of the expected 06:00 Swedish time.

I'm currently trying this out on Vercel and running the `quirrel ci` command to register the jobs.
Have I missunderstod something or is this a bug?

Thanks for a awesome library, really like it a lot so far!

**Comment by Skn0tt** [comment, marked as answer]
You've discovered a bug, thank you for reporting this! I've opened an issue to track it here: #835
